chapters/solutions.Rmd

```diff
@@ -206,15 +206,22 @@ The file should contain the following lines:
 
 ### Exercise \@ref(bash-basics-ex-pipe-construction) {-}
 
-```shell
-$ cut -d , -f 2 animals.txt | sort | uniq
-```
+`cut` selects substrings from a line by:
+
+-   breaking the string into pieces wherever it finds a separator (`-d ,`) and
```

Comment:
add "in this case, a comma" to disambiguate the parameter from potential syntax associated with unix